tools/pkg/scripts/smoke_test.sh

```diff
@@ -5,6 +5,15 @@
 set -eu pipefail
 IFS=$'\n\t'
 
+echo "Check, that print_install_dir works"
+MIM_DIR=$(mongooseimctl print_install_dir)
+test -d "$MIM_DIR"
+
+# Template generation needs writable etc directory, but mongooseimctl switches user,
+# so it's not executed as root.
+chmod 777 "$MIM_DIR/etc"
```

Comment:
@arcusfelis @sebb7 I have the impression that it would be better to set the permission to the `etc` in some other places when building or installing the package and not when running scripts.

Reply:
I think that we should not set any permissions in smoke tests. I thought that smoke tests had to check some commands which would be run by the potential package user after installing it.

Reply:
We can set mongooseim user to be owner of the configs, so he can change the configs.
Though it depends if we want the user to change configs or not.